lms/djangoapps/course_home_api/toggles.py

```diff
@@ -6,6 +6,15 @@
 
 WAFFLE_FLAG_NAMESPACE = 'course_home'
 
+# Waffle flag to enable amplitude recommendations
+# .. toggle_name: course_home.course_home_mfe_progress_tab
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: toggles if the LMS progress tab UI should use the new Learning MFE graphs or
+#   grades graphs from edx-platform.
+# .. toggle_use_cases: opt_in
+# .. toggle_creation_date: 2021-03-12
+# .. toggle_tickets: https://github.com/openedx/edx-platform/pull/26978
 COURSE_HOME_MICROFRONTEND_PROGRESS_TAB = CourseWaffleFlag(  # lint-amnesty, pylint: disable=toggle-missing-annotation
```

Comment:
You can remove `# lint-amnesty, pylint: disable=toggle-missing-annotation`, because we are adding the annotation.